### ISSUE TITLE
单个设备预测时权重文件与模型结构不匹配

### ISSUE THREAD

**Issue by WXX740813** [ChanceQZ/RoofTopSegmatation#2]
作者您好，请问我在使用您的预训练模型的时候，当我将BatchNorm2d = SynchronizedBatchNorm2d改为BatchNorm2d = nn.BatchNorm2d后，使用您已经训练好的权重文件加载deeplab_xception文件中的DeepLabv3_plus模型框架，进行预测时，报权重与模型结构不匹配。请问这是什么原因呢。
